plugins/convert_README_to_header.sh

```diff
@@ -30,8 +30,14 @@ outfilename=`basename $outfile`;
 # See http://stackoverflow.com/a/16576291
 # On Mac OS's sed, \n is not recognized as a newline character, but
 # \[actual newline] works
-desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\"/\\\"/g' -e 's/\n/\\\\n"\\
-"/g' "$path/README.md"`;
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\\\/\\\\\\\/g' -e 's/\"/\\\"/g'\
+ -e 's/\n/\\\\n\"\
+\"/g' "$path/README.md"`;
+
+if (( $? != 0 )); then
+	echo 'Sed failed to generate $desc'
```

Comment:
In another PR please, I suspect returning the desc might not be helpful as a flood of text, unless you redirect stderr to stdout too, which has its own risks.

Similar stuff what's doing to all the other backticks now we know of the issue.

Reply:
It's not returning the content fo $desc, single quotes don't resolve variables.